flake.nix

```diff
@@ -13,7 +13,7 @@
   inputs = {
     nixpkgs.url = "github:NixOS/nixpkgs/nixpkgs-unstable";
     rustowl = {
-      url = "github:cordx56/rustowl";
+      url = "github:cordx56/rustowl?ref=v0.3.0";
```

Comment:
issue: we shouldn't pin the flake input - that will break automatic updates. 
If an automatic flake.lock update fails, we can either fix the build or, if it's an upsteam issue, hold off on merging until it has been fixed upstream.